spec/features/admin_add_superpowers_spec.rb

```diff
@@ -1,9 +1,9 @@
 require 'rails_helper'
 
-feature 'user can add superpowers' do
+feature 'admin can add superpowers' do
```

Comment:
Do you ONLY want admins to be able to create superpowers?

If all users should be able to create superpowers, you definitely want to remove the "admin" aspect from this test to make sure all users can create.

Reply:
Yes, only admins can create superpowers